Fleece/Support/endianness.h

```diff
@@ -130,7 +129,7 @@
 #  define ntoh64(x)     (x)
 #  define hton64(x)     (x)
 #  else
-#    warning "UNKNOWN Platform / endianness; network / host byte swaps not defined."
+#   pragma message ( "UNKNOWN Platform / endianness; network / host byte swaps not defined.")
```

Comment:
What is the purpose of changing this?

Reply:
MSVC doesn't support #warning directive, it issues a `C1021: invalid preprocessor command 'warning'` error. #pragma message is supported by MSVC, clang, gcc compilers.

Reply:
Wow so it is!  I swore it worked in MSVC too but in fact it doesn't.  Let's add "warning:" to the beginning of the message.  I find that just emitting a message won't show anything in the errors / warnings tab of VS but adding "warning: " to the beginning will make it into a warning.  That's very unfortunate since now on both compilers you will get an entry with no line number or any sort of information other than maybe the file it was in....this seems like a rare enough case though so

Reply:
Done